crates/bevy_ecs/src/archetype.rs

```diff
@@ -532,6 +532,13 @@ impl Archetype {
             .map(|info| info.archetype_component_id)
     }
 
+    pub fn shrink_to_fit(&mut self) {
```

Comment:
This should probably be `pub(crate)`. There's no way for a user of `bevy_ecs` to get a `&mut Archetype` without being unsound.